admin/lib/validator/check-form/single-check-form-validator.js

```diff
@@ -40,7 +40,7 @@ singleCheckFormValidator.validate = async (uploadedFile) => {
   const fileContent = fileBuffer && fileBuffer.toString().trim()
   const fileLines = fileContent && fileContent.split('\n').length
   // Invalid total file lines
-  if (fileLines !== config.LINES_PER_CHECK_FORM) {
+  if (fileLines !== parseInt(config.LINES_PER_CHECK_FORM)) {
```

Comment:
parseInt() should still be used with a  radix, because `parseInt("010")` can either be octal or decimal depending on the implementation, and it's always possible implementation can change.

```suggestion
  if (fileLines !== parseInt(config.LINES_PER_CHECK_FORM, 10)) {
```

Same for all the other uses.